src/components/network-chart/index.js

```diff
@@ -60,8 +75,21 @@ class NetworkChart extends React.PureComponent<Props> {
     // Not implemented.
   };
 
+  _onRightClick = (markerIndex: MarkerIndex) => {
+    const { threadIndex, changeRightClickedMarker } = this.props;
+    changeRightClickedMarker(threadIndex, markerIndex);
+  };
+
+  _shouldDisplayTooltips = () => this.props.rightClickedMarkerIndex === null;
```

Comment:
implementing this using `rightClickedMarkerIndex` directly instead of `isContextMenuVisible` brings some advantages:
* it's simpler
* it solved another problem where the previously displayed tooltip was still shown after right clicking => because when the row was rerendering because the `rightClickedMarkerIndex` value changed, `isContextMenuVisible` was still `false`, and as a result the tooltip was still rendered. Doing it this way makes it work as expected.

I filed https://github.com/firefox-devtools/profiler/issues/2064 to try to do it the same way in other panels too.